v3/integration/lints/lint/lint.go

```diff
@@ -0,0 +1,199 @@
+package lint
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+type Lint interface {
+	Lint(tree *ast.File, file *File) *Result
+	CheckApplies(tree *ast.File, file *File) bool
+}
+
+// A Result encodes any unmet expectation laid out by your lint. It consists of a single message, a list of code
+// citations, and a list of lint citations.
+//
+// The message should be succinct and descriptive of the core issue. This message can only be set in the constructor,
+// NewResult. For example...
+//
+//		"Go style guides suggest not using bare returns in complex functions"
+//
+// Code citations are the locations within the file that did not meet your expectations. Please see AddCodeCitations
+// for information on how to add these to the Result type. Adding a code citation will result in the file, line number
+// and raw source code appearing in the lint result. For example...
+//
+//	File ../../lints/cabf_br/lint_cab_dv_conflicts_with_locality.go, line 28
+//
+//	func (l *certPolicyConflictsWithLocality) Initialize() error {
+//		return nil
+//	}
+//
+// The lint citations are additional information to help the contributor understand why their code failed
+// this lint and, if possible, some hints or resources on how to correct the issue. Every citation will listed on its
```

Comment:
typo
```suggestion
// this lint and, if possible, some hints or resources on how to correct the issue. Every citation will be listed on its
```

Reply:
I think this typo's still unresolved..

Reply:
Sorry, must've missed it - certainly not skipping out of conversations.